Sources/App/Logger/LogLevel+Compare.swift

```diff
@@ -0,0 +1,33 @@
+import Logging
+
+extension LogLevel: Equatable {
+    public static func == (lhs: LogLevel, rhs: LogLevel) -> Bool {
+        return lhs.description == rhs.description
+    }
+
+    /// From more verbose to less verbose (i.e. from less important to more important)
+    private static let allLevels: [LogLevel] = [
+        .verbose, .debug, .info, .warning, .error, .fatal
+    ]
+
+    // Note: Didn't implement it as Comparable and `<` operator because `.custom` case doesn't follow semantics.
+    public func isAtLeast(minimumLevel: LogLevel) -> Bool {
+        // If minimumLevel is .custom, only keep the ones that are exactly using the same custom level name
+        if case .custom(let minValue) = minimumLevel, case .custom(let selfValue) = self {
+            return selfValue == minValue
+        }
+        // Otherwise, consider custom levels have the same "importance" as .info
+        let defaultIndex = LogLevel.allLevels.firstIndex(of: .info)!
+        let minIndex = LogLevel.allLevels.firstIndex(of: minimumLevel) ?? defaultIndex
```

Comment:
How can `minimumLevel` not exist in this context? Also assuming a default in here doesn't seem right anyway.

Reply:
As expressed by the comment, it's for for the case when `minimumLevel` is `.custom("Foo")`.

(`allLevels` might be a misleading name, as it doesn't contain all levels but all pre-defined levels. naming is hard.)